nautobot/docs/release-notes/version-1.2.md

```diff
@@ -140,6 +140,27 @@ All models that have `slug` fields now use `AutoSlugField` from the `django-exte
 
 Just as with the UI, the `slug` can still always be explicitly set if desired.
 
+## v1.2.6 (2022-MM-DD)
+
+### Added
+
+- [#1353](https://github.com/nautobot/nautobot/issues/1353) - Added UI for deleting previously uploaded images when editing a DeviceType.
+
+### Fixed
+
+- [#1249](https://github.com/nautobot/nautobot/issues/1249) - Fixed a timing issue where after creating a custom field with a default value and immediately assigning values to this custom field on individual objects, the custom field values could be automatically reverted to the default value.
+- [#1279](https://github.com/nautobot/nautobot/issues/1279) - Circuit terminations now render custom relationships on the circuit detail page.
```

Comment:
Possibly move this item up under `Added` instead? Debatable.

Reply:
Yeah I would say that is an `Added`.